astropy/table/tests/test_masked.py

```diff
@@ -566,18 +567,32 @@ def test_masked_as_array_with_mixin():
 
 
 def test_masked_column_with_unit_in_qtable():
-    """Test that adding a MaskedColumn with a unit to QTable issues warning"""
+    """Test that adding a MaskedColumn with a unit to QTable creates a MaskedQuantity."""
+    MaskedQuantity = Masked(u.Quantity)
```

Comment:
This relates to the API question above, namely that I foresee this boilerplate line occurring a lot.